scripts/utils/generate_strings.py

```diff
@@ -238,18 +276,25 @@ def serialize(string):
     )
     args = parser.parse_args()
 
-    if not args.source:
+    if not args.sources:
         exit("No source argument.")
 
     # If no output directory was provided, use the current directory.
     if args.output is None:
         args.output = os.getcwd()
 
-    # Parse the inputs for their sweet juicy strings.
```

Comment:
🙄

Reply:
Heh, I was hoping I could just quietly remove this.